packages/ui/src/components/library/Autocomplete.tsx

```diff
@@ -0,0 +1,141 @@
+import { Autocomplete as AutocompleteMui } from '@mui/material'
+import React, { SyntheticEvent } from 'react'
+import { styled } from '@mui/material/styles'
+import { AutocompleteRenderInputParams } from '@mui/material/Autocomplete/Autocomplete'
+import { HiOutlineChevronDown } from 'react-icons/hi2'
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types'
+
+interface AutocompleteProps {
+  InputProps?: Partial<AutocompleteRenderInputParams['InputProps']>
+  className?: string
+  isOptionEqualToValue?: (option: any, value: any) => boolean
+  filterOptions?: (options: any[], state: any) => any[]
+  options: any[]
+  getOptionLabel: (option: any) => string
+  onChange?: (event: any, value: any) => void
+  value?: any
+  renderOption: (props: any, option: any, state: any) => React.ReactNode
+  disableClearable?: boolean
+  onKeyDown?: (e: any) => void
+  onInputChange?: (
+    _: SyntheticEvent<Element, Event>,
+    val: string | InjectedAccountWithMeta | null
+  ) => void
+  disabled?: boolean
+  freeSolo?: boolean
+  renderInput: (params: AutocompleteRenderInputParams) => React.ReactNode
+  iconSize?: string
+  selectOnFocus?: boolean
+  clearOnBlur?: boolean
+  handleHomeEndKeys?: boolean
+}
+
+const Autocomplete = ({
+  className,
+  isOptionEqualToValue,
+  filterOptions,
+  options,
+  getOptionLabel,
+  onChange,
+  value,
+  renderOption,
+  disableClearable,
+  disabled,
+  freeSolo,
+  onInputChange,
+  renderInput,
+  selectOnFocus,
+  clearOnBlur,
+  handleHomeEndKeys,
+  iconSize
+}: AutocompleteProps) => {
+  return (
+    <AutocompleteWrapper
+      iconSize={iconSize}
+      className={className}
+    >
+      <AutocompleteMui
+        handleHomeEndKeys={handleHomeEndKeys}
+        selectOnFocus={selectOnFocus}
+        clearOnBlur={clearOnBlur}
+        disablePortal
+        isOptionEqualToValue={isOptionEqualToValue}
+        disableClearable={disableClearable}
+        filterOptions={filterOptions}
+        options={options}
+        getOptionLabel={getOptionLabel}
+        onChange={onChange}
+        value={value}
+        disabled={disabled}
+        freeSolo={freeSolo}
+        onInputChange={onInputChange}
+        renderOption={renderOption}
+        popupIcon={<HiOutlineChevronDown />}
+        renderInput={renderInput}
+      />
+    </AutocompleteWrapper>
+  )
+}
+
+Autocomplete.defaultProps = {
+  iconSize: '1.25rem'
```

Comment:
I find the icons way too small in a dropdown, and don't really display nicely with 2 information such as a name and address.
Identicons are a way to identify accounts quickly and need to be large enough for a selection I'd say. Ping @sweetpea22 

![image](https://github.com/ChainSafe/Multix/assets/33178835/b6563ae6-fb9e-4c01-978d-4a3dfd1cf8e2)

Reply:
yes it looks like my design here is wrong, it's too small to look good. I'm thinking to only use the big ones like this and remove the address in the dropdown for tidiness. but i'll give it a go to see how this looks first, brb
<img width="185" alt="Screen Shot 2023-07-11 at 11 59 46 AM" src="https://github.com/ChainSafe/Multix/assets/47398578/517fb8fd-11c3-42e1-b74c-acdfc75e2b00">
<img width="499" alt="Screen Shot 2023-07-11 at 12 00 11 PM" src="https://github.com/ChainSafe/Multix/assets/47398578/9bc8025b-0595-490d-90aa-55ca07223bb7">

Reply:
agreed, cool